src/main/java/io/ipinfo/spring/strategies/ip/XForwardedFor.java

```diff
@@ -0,0 +1,20 @@
+package io.ipinfo.spring.strategies.ip;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class XForwardedFor implements IPStrategy {
+    /**
+     * Gets the ip address using the X-Forwarded-For header.
+     *
+     * @param request originated from a client to this server.
+     * @return
+     */
+    @Override
+    public String getIPAddress(HttpServletRequest request) {
+        String XforwardedFor = request.getHeader("X-Forwarded-For");
```

Comment:
I'm not sure myself, but does this lowercase the headers properly? Like does it handle mixed case on its own?

Reply:
Also the variable should be lowercase, `xForwardedFor`, I think.

Reply:
Yes the the function is case insensitive so it'll handle all it properly